aspnetcore/fundamentals/logging/index.md

```diff
@@ -13,15 +13,11 @@ uid: fundamentals/logging/index
 
 ::: moniker range=">= aspnetcore-3.0"
 
-By [Kirk Larkin](https://twitter.com/serpent5), [Juergen Gutsch](https://github.com/JuergenGutsch) and [Rick Anderson](https://twitter.com/RickAndMSFT)
+By [Kirk Larkin](https://twitter.com/serpent5), [Juergen Gutsch](https://github.com/JuergenGutsch), and [Rick Anderson](https://twitter.com/RickAndMSFT)
```

Comment:
Do you want me to make any other changes here? I saw there's an issue about maybe dropping these or stripping them down a bit... I can make the changes while I've got this open.

Reply:
Not now. Later on we'll remove FTE's only, so your name would stay/